New models

JMGC34 opened the pull request:
Having issues with matplotlib.pyplot.savefig() so for now the models have been manually saved as an old .doc file.  Still figuring out HDP.  Let me know what you think of the graphs...how you might want them to be labeled, colored, etc...

JMGC34 committed:
Add files via upload

JMGC34 committed:
Delete HDP_model

JMGC34 committed:
Delete LDA_model

JMGC34 committed:
Delete LDA_model.expElogbeta.npy

JMGC34 committed:
Delete LDA_model.id2word

JMGC34 committed:
Delete LDA_model.state

JMGC34 committed:
Delete LSI_model

JMGC34 committed:
Delete LSI_model.projection

JMGC34 committed:
Delete Topic_model_results.docx

Inquisitive-Geek commented:
What topics do you think can be interpreted from the model results? Can you rename the nodes in the visualization to explain a bit more about them?

JMGC34 committed:
Add files via upload

List of topics from a variety of model outputs.  Some of them are repeats, but if people could take a look and comment with any ones they think are in the right direction, that would be helpful.  Use the model name and ordered pair to identify please.

Inquisitive-Geek reviewed (commented):
Great work
